### ISSUE TITLE
Cannot launch streams anymore after having updated to Streamlink v3.1.0

### ISSUE THREAD

**Issue by Robby-** [streamlink/streamlink-twitch-gui#855]
### Checklist

- [X] I understand that **Streamlink Twitch GUI** is just a launcher for **Streamlink**
- [X] [I have read the contribution guidelines](https://github.com/streamlink/streamlink-twitch-gui/blob/master/CONTRIBUTING.md)
- [X] [I have checked the list of open and recently closed issues](https://github.com/streamlink/streamlink-twitch-gui/issues?q=is%3Aissue+sort%3Aupdated-desc)
- [X] [I have checked the commit log of the master branch](https://github.com/streamlink/streamlink-twitch-gui/commits/master)

### Streamlink Twitch GUI version

v1.13.0

### Streamlink version

3.1.0

### Operating system, environment and configuration details

Windows 10 21H1

### Description

After having updated Streamlink to version 3.1.0, the Streamlink Twitch GUI cannot launch streams anymore.
When clicking a stream, a popup immediately shows up with an error. The error displayed in this popup is:

```
Error
Exit code 3221225477
```

Prior to this I was using Streamlink 2.4.0, so I can't really say if this issue started with version 3.0.0 or a later version.

### Debug log

```text
[2022-01-24T17:08:34.277Z][debug][Application]
Parameters
{
    "_": [],
    "tray": false,
    "hide": false,
    "hidden": false,
    "max": false,
    "maximize": false,
    "maximized": false,
    "min": false,
    "minimize": false,
    "minimized": false,
    "reset-window": false,
    "versioncheck": true,
    "version-check": true,
    "logfile": true,
    "loglevel": "debug",
    "l": "debug",
    "launch": "",
    "goto": ""
}

[2022-01-24T17:08:37.735Z][debug][StreamingService]
Preparing to launch stream
{
    "id": "sodapoppin",
    "quality": "source",
    "low_latency": true,
    "disable_ads": true,
    "chat_open": false,
    "started": "2022-01-24T17:08:37.710Z",
    "stream": "26301881",
    "channel": "26301881"
}

[2022-01-24T17:08:37.742Z][debug][StreamingService]
Resolving streaming provider
{
    "provider": "streamlink",
    "providerUserData": {
        "exec": "",
        "params": "",
        "pythonscript": null
    }
}

[2022-01-24T17:08:37.745Z][debug][StreamingService]
Found streaming provider
{
    "exec": "C:\\Program Files (x86)\\Streamlink\\bin\\streamlinkw.exe",
    "params": null,
    "env": null
}

[2022-01-24T17:08:37.790Z][debug][StreamingService]
Spawning process
{
    "exec": "C:\\Program Files (x86)\\Streamlink\\bin\\streamlinkw.exe",
    "params": [
        "--version"
    ],
    "env": null
}

[2022-01-24T17:08:37.848Z][error][StreamingService]
Error: Exit code 3221225477
{
    "id": "sodapoppin",
    "quality": "source",
    "low_latency": true,
    "disable_ads": true,
    "chat_open": false,
    "started": "2022-01-24T17:08:37.710Z",
    "stream": "26301881",
    "channel": "26301881"
}
```

**Comment by bastimeyer** [comment]
https://github.com/streamlink/streamlink/issues/4307

**Comment by bastimeyer** [comment]
Fixed in Streamlink 3.1.1
https://github.com/streamlink/streamlink/releases/tag/3.1.1
